nmdc_runtime/api/main.py

```diff
@@ -510,9 +545,67 @@ def custom_swagger_ui_html(
         .replace('</unquote-safe>"', "")
         .replace("<double-quote>", '"')
         .replace("</double-quote>", '"')
+        # TODO: Remove unnecessary trailing backslashes.
         .replace(
             "</head>",
-            "<style>.nmdc-info-banner { padding: 1em; background-color: #448aff1a; border: .075rem solid #448aff; }</style></head>",
+            f"""<style>\
+                    .nmdc-info {{\
+                        padding: 1em;\
+                        background-color: #448aff1a;\
+                        border: .075rem solid #448aff;\
+                    }}\
+                    .nmdc-info-token code {{\
+                        font-size: x-small;\
+                    }}\
+                </style>\
+            </head>""",
```

Comment:
I didn't realize this snippet was added as part of this same PR. I will remove the unnecessary trailing backslashes now.